Source/towners.h

```diff
@@ -1,76 +1,77 @@
-/**

```

Comment:
Please configure your editor to respect line endings in `.editorconfig`

Reply:
Github messed up this file when I resolved a conflict in the browser.